packages/itwinui-css/src/style/table-densities.scss

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 // See LICENSE.md in the project root for license terms and full copyright notice.
-@import '../style/index';
+@import './baseline';
+@import './space';
 
 $iui-table-density-vertical: $iui-baseline * 1.5;
```

Comment:
hm... I was thinking, maybe it makes more sense to move this together with table? And all other files, if they are specific to certain component.. No point to move those to generic styles

Reply:
That makes sense for table-densities and avatar-sizes, but notification-marker is used in multiple places, including button, side-navigation, and header.